Documentation/ceph-object-multisite.md

```diff
@@ -28,20 +29,20 @@ This guide assumes a Rook cluster as explained in the [Quickstart](quickstart.md
 
 If an admin wants to set up multisite on a Rook Ceph cluster, the admin should create:
 
-1. A [realm](/Documentation/ceph-object-multisite-crd.md#object-realm-settings)
-1. A [zonegroup](/Documentation/ceph-object-multisite-crd.md#object-zone-group-settings)
-1. A [zone](/Documentation/ceph-object-multisite-crd.md#object-zone-settings)
-1. An [object-store](/Documentation/ceph-object-store-crd.md#zone-settings) with the `zone` section
+1. A [realm](ceph-object-multisite-crd.md#object-realm-settings)
+1. A [zonegroup](ceph-object-multisite-crd.md#object-zone-group-settings)
+1. A [zone](ceph-object-multisite-crd.md#object-zone-settings)
+1. An [object-store](ceph-object-store-crd.md#zone-settings) with the `zone` section
 
-object-multisite.yaml in the [examples](/deploy/examples/) directory can be used to create the multisite CRDs.
+object-multisite.yaml in the [examples](https://github.com/rook/rook/blob/{{ branchName }}/deploy/examples/) directory can be used to create the multisite CRDs.
 
 ```console
 kubectl create -f object-multisite.yaml
 ```
 
 The first zone group created in a realm is the master zone group. The first zone created in a zone group is the master zone.
 
-When a non-master zone or non-master zone group is created, the zone group or zone is not in the Ceph Radosgw Multisite [Period](https://docs.ceph.com/docs/master/radosgw/multisite/) until an object-store is created in that zone (and zone group).
+When a non-master zone or non-master zone group is created, the zone group or zone is not in the Ceph Radosgw Multisite [Period](https://docs.ceph.com/docs/{{ branchName }}/radosgw/multisite/) until an object-store is created in that zone (and zone group).
```

Comment:
For the ceph docs we just link the latest, rather than a branch name.
```suggestion
When a non-master zone or non-master zone group is created, the zone group or zone is not in the Ceph Radosgw Multisite [Period](https://docs.ceph.com/en/latest/radosgw/multisite/) until an object-store is created in that zone (and zone group).
```